docs/content/any/project/changelogs/2021-05-17.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Release 2021-05-17
+menuTitle: 2021-05-17
+any: true
+description: >-
+  Changelog for Release 2021-05-17 (sqlalchemy-oso 0.6.2, sqlalchemy-oso-preview
+  0.0.3) containing new features, bug fixes, and more.
+---
+
+## `sqlalchemy-oso` 0.6.2
+
+### Other bugs & improvements
+
+- Authorized sessions now disable [baked queries][] by default because the
+  caching mechanism can bypass authorization by using queries from the cache
+  that were previously baked without authorization applied. If you understand
+  the risks and still want to opt-in to the previous behavior of using baked
+  queries, you can pass the `enable_baked_queries=True` keyword argument to
+  `sqlalchemy_oso.authorized_sessionmaker()` and friends.
```

Comment:
Have the docs been updated?

Reply:
Which docs? I updated doc strings so the sqlalchemy-oso API docs will reflect this change.

Reply:
I don't think `enable_baked_queries=True` needs to be updated anywhere b/c folks shouldn't use it except in very niche use cases. It'll be in the API docs for anyone who goes looking for it.